src/test/java/reposense/util/TimeUtilTest.java

```diff
@@ -9,7 +9,7 @@
 
 public class TimeUtilTest {
     @Test
-    public void extractDate_validDate_success() throws Exception {
+    public void extractDate_validDate_success() {
```

Comment:
I think you missed out on `extractDate_validDateAndTime_success()`.

Reply:
For some reason IntelliJ does not indicate that is is redundant as shown in the image below:
![Screen Shot 2021-07-17 at 2 21 20 PM](https://user-images.githubusercontent.com/61911161/126027956-0de6b9ab-1109-4f3d-be72-051944cac8f3.png)

However, it can be removed without incurring a compilation error either. It makes sense as both test cases use the same set of methods internally. Thanks for pointing this out!